vocabularies/qld-mining-permits.ttl

```diff
@@ -0,0 +1,305 @@
+    skos:altLabel "Geothermal permits"@en ;
+    skos:definition "An appropriate resource authority is required before you can conduct any exploration or production activity for geothermal energy in Queensland. These are issued under Queensland's Geothermal Energy Act 2010."@en ;
+    skos:hiddenLabel "Geothermal tenures"@en ;
+    skos:member minpermit:ExplorationPermitGeothermal,
+        minpermit:GeothermalLease,
+        minpermit:PotentialGeothermalCommercialArea ;
+    skos:prefLabel "Geothermal authorities"@en .
+
+minpermit:GreenhouseGasAuthorities a skos:Collection ;
+    skos:definition "An appropriate resource authority is required before you can conduct any exploration or production activity for greenhouse gas storage areas in Queensland. These are issued under Queensland's Greenhouse Gas Storage Act 2009."@en ;
+    skos:member minpermit:DataAcquisitionAuthority,
+        minpermit:ExplorationPermitGreenhouseGas,
+        minpermit:GreenhouseGasLease,
+        minpermit:PotentialGreenhouseGasStorageCommercialArea ;
+    skos:prefLabel "Greenhouse gas storage authorities"@en .
+
+minpermit:MiningAuthorities a skos:Collection ;
+    skos:altLabel "Mining permits"@en,
+        "Mining tenures"@en ;
+    skos:hiddenLabel "Mining tenements"@en ;
+    skos:member minpermit:ExplorationPermitCoal,
+        minpermit:ExplorationPermitMineral,
+        minpermit:ExplorationPermitSpecial,
+        minpermit:MineralDevelopmentLicence,
+        minpermit:MiningClaim,
+        minpermit:MiningLease,
+        minpermit:WaterMonitoringAuthority ;
+    skos:prefLabel "Mining authorities"@en .
+
+minpermit:PetroleumAndGasAuthority a skos:Collection ;
+    skos:altLabel "Petroleum and gas permit"@en ;
+    skos:definition """An appropriate resource authority is required before you can conduct any exploration or production activity for petroleum and gas in Queensland. These are issued under Queensland's Petroleum and Gas (Production and Safety) Act 2004. 
+
+Petroleum and gas authorities are issued for: 
+
+Exploration - authority to prospect, potential commercial area and petroleum lease 
+
+Production - petroleum lease 
+
+Infrastructure development - petroleum facility licence and petroleum pipeline licence 
+
+Information gathering - petroleum survey licence, water monitoring authority and data acquisition authority."""@en ;
+    skos:member minpermit:AuthorityToProspect,
+        minpermit:DataAcquisitionAuthority,
+        minpermit:PetroleumFacilityLicence,
+        minpermit:PetroleumLease,
+        minpermit:PetroleumPipelineLicence,
+        minpermit:PetroleumSurveyLicence,
+        minpermit:PotentialCommercialArea,
+        minpermit:WaterMonitoringAuthority ;
+    skos:prefLabel "Petroleum and gas resource authority"@en .
+
+minpermit:ProspectingPermit a skos:Concept ;
+    skos:definition """A prospecting permit can be sought for any mineral other than coal. There are 2 types of prospecting permit:
```

Comment:
triple quote marks at start and end of definition

Reply:
https://www.w3.org/TR/sparql11-query/#QSynLiterals

To facilitate writing literal values which themselves contain quotation marks or which are long and contain newline characters, SPARQL provides an additional quoting construct in which literals are enclosed in three single- or double-quotation marks.

The lines affected contain the newline character \n